coffea/jetmet_tools/FactorizedJetCorrector.py

```diff
@@ -2,8 +2,10 @@
 import warnings
 import re
 from ..util import awkward
+from ..util import awkward1
```

Comment:
I think we can drop the `from util import X` now that the whole `cupy/numpy` switch business is done in another way.